src/lib/WaveIcon.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang="ts">
+ import type { WaveType } from '$lib/audio/instrument';
+ export let wave: WaveType;
+ export let selected: boolean;
+</script>
+
+<svg
+    width="200mm"
+    height="200mm"
```

Comment:
Remove width and height, they do nothing here because you override them later in css.

Reply:
done